### ISSUE TITLE
Video Call issue

### ISSUE THREAD

**Issue by HaiderMM** [EricssonResearch/bowser#88]
Hi,

Bowser stuck, when receive/dial a video call on iPhone.
And also I am not able to see the console. Is there any way to resolve this issue or to see the console on iOS.

thanks in advance.

regards,
Haider Ali Khan

**Comment by stefanalund** [comment]
The best way to see the console is to connect an iOS device to Xcode and use this trick:

http://moduscreate.com/enable-remote-web-inspector-in-ios-6/

**Comment by logidelic** [comment]
@stefanalund Are you suggesting that this method can be used to see Bowser's console on a Mac? I followed these steps and it works for Safari, but I don't see how to connect to the open Bowser page.

I also tried viewing the "device console" in Xcode, but it doesn't seem to include console.log() stuff or much else from Bowser. Should it?

Thanks much.

**Comment by stefanalund** [comment]
@logidelic yes, you should be able to see the web inspector if you launch and run Bowser from within Xcode.
